app/src/pages/inside/testCaseLibraryPage/historyOfActions/historyOfActions.scss

```
@@ -49,13 +148,40 @@
     }
   }
 
+  &__table-cell-value {
+    display: block;
+    box-sizing: border-box;
+    width: 100%;
+    min-width: 0;
+    max-width: 100%;
+    overflow: visible;
+    white-space: normal;
+    word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)